eZ/Bundle/EzPublishCoreBundle/EventListener/RequestEventListener.php

```diff
@@ -76,9 +76,11 @@ public function onKernelRequestIndex( GetResponseEvent $event )
     {
         $request = $event->getRequest();
         $semanticPathinfo = $request->attributes->get( 'semanticPathinfo' ) ?: '/';
+        $isLegacyMode = $this->configResolver->getParameter( 'legacy_mode', 'ezsettings' );
```

Comment:
@lolautruche How are we supposed to handle this mode now that legacy has been moved out?